src/commands/addons/addons.ts

```diff
@@ -51,7 +51,7 @@ Add-ons are a way to extend the functionality of your Netlify site`,
     .description(
       `Remove an add-on extension to your site\nAdd-ons are a way to extend the functionality of your Netlify site`,
     )
-    .option('-f, --force', 'delete without prompting (useful for CI)')
+    // .option('-f, --force', 'delete without prompting (useful for CI)')
```

Comment:
Why is this commented out?

Reply:
It was left over, it was there as a placeholder.